src/serviceFacades/apps.js

```diff
@@ -38,23 +39,74 @@ function getAppById(key, { systemId, appId }) {
     });
 }
 
-function getPrivateCategories(key) {
-    return callApi({
-        endpoint: "/api/apps/categories?public=false",
-        method: "GET",
-    });
+function getPrivateCategories(key, userId) {
+    return userId
+        ? callApi({
+              endpoint: "/api/apps/categories?public=false",
+              method: "GET",
+          })
+        : Promise.resolve({
+              categories: [
+                  {
+                      system_id: "de",
+                      name: "Workspace",
+                      categories: [
+                          {
+                              system_id: "de",
+                              id: constants.APPS_UNDER_DEV,
+                              name: constants.APPS_UNDER_DEV,
+                          },
+                          {
+                              system_id: "de",
+                              id: constants.FAV_APPS,
```

Comment:
Here's an example of a fake category ID.  For `Favorite Apps` and `Apps Under Development` the category IDs are unique for each user and I'm setting a fake ID for the anonymous user.

Reply:
"setting a fake ID for the anonymous user" might be worth adding as a comment to this source somewhere.